model/gen-ai/spans.yaml

```
@@ -275,6 +284,12 @@ groups:
       - ref: gen_ai.data_source.id
         requirement_level:
           conditionally_required: if applicable.
+      - ref: server.address
+        requirement_level:
+          recommended: when span kind is `CLIENT`.
+      - ref: server.port
+        requirement_level:
+          conditionally_required: when span kind is `CLIENT` and `server.address` is set.
```

nit: it's not really necessary to add `CLIENT` to this condition, server.address is enought

```suggestion
          conditionally_required: If `server.address` is set.
```

or remove `requirement_level` - it's inherited

---

makes sense - removed the override so it will just inherit from common attributes